xdsl/dialects/experimental/fir.py

```diff
@@ -1142,8 +1143,12 @@ class DispatchTable(IRDLOperation):
     """
 
     name = "fir.dispatch_table"
+
+    sym_name = attr_def(StringAttr)
```

Comment:
I guess that in the tests, `sym_name` was used but not required?

Reply:
I don't know, I'm not sure it's tested tbh! I chatted with @mesham and ensure this won't be an issue short term either way 🙂